packages/amplication-server/src/core/prismaSchemaParser/schema-utils.ts

```diff
@@ -404,14 +412,13 @@ export function handleModelNamesCollision(
   let counter = 0;
 
   do {
+    isFormattedModelNameAlreadyTaken ||=
```

Comment:
I think you have a bug here with using the ||= before you assign it directly in line 418